common/src/main/java/org/apache/nemo/common/dag/DAGBuilder.java

```diff
@@ -169,6 +176,94 @@ public DAGBuilder(final DAG<V, E> dag) {
     return this;
   }
 
+  /**
+   * Use this method when there are no edges to be replaced.
```

Comment:
replaced by?

Reply:
I've added more detailed explanations about this method. Could you please check it? Thanks.